python-squid/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -726,61 +727,61 @@ public void nonlocalStatement() {
   public void funcdef_statement() {
     setRootRule(PythonGrammar.FUNCDEF);
     AstNode astNode = p.parse("def func(): pass");
-    FunctionDef functionDefTree = treeMaker.funcDefStatement(astNode);
-    assertThat(functionDefTree.name()).isNotNull();
-    assertThat(functionDefTree.name().name()).isEqualTo("func");
-    assertThat(functionDefTree.body().statements()).hasSize(1);
-    assertThat(functionDefTree.body().statements().get(0).is(Tree.Kind.PASS_STMT)).isTrue();
-    assertThat(functionDefTree.children()).hasSize(6);
-    assertThat(functionDefTree.parameters()).isNull();
-    assertThat(functionDefTree.isMethodDefinition()).isFalse();
-    assertThat(functionDefTree.docstring()).isNull();
-    assertThat(functionDefTree.decorators()).isEmpty();
-    assertThat(functionDefTree.asyncKeyword()).isNull();
-    assertThat(functionDefTree.returnTypeAnnotation()).isNull();
-    assertThat(functionDefTree.colon().value()).isEqualTo(":");
-    assertThat(functionDefTree.defKeyword().value()).isEqualTo("def");
-    assertThat(functionDefTree.leftPar().value()).isEqualTo("(");
-    assertThat(functionDefTree.rightPar().value()).isEqualTo(")");
-
-    functionDefTree = parse("def func(x): pass", treeMaker::funcDefStatement);
-    assertThat(functionDefTree.parameters().all()).hasSize(1);
-
-    functionDefTree = parse("async def func(x): pass", treeMaker::funcDefStatement);
-    assertThat(functionDefTree.asyncKeyword().value()).isEqualTo("async");
-
-    functionDefTree = parse("def func(x) -> string : pass", treeMaker::funcDefStatement);
-    TypeAnnotation returnType = functionDefTree.returnTypeAnnotation();
+    FunctionDef functionDef = treeMaker.funcDefStatement(astNode);
```

Comment:
This is great to rename the variable, but, as this is unrelated to the task at hand could you try to keep it in a separate commit next time as it makes things hard to review : this is tough to separate refactoring from required change at review time. (no action required for this comment).

Reply:
Indeed! Will make separate commits next time :)